.gitignore

```diff
@@ -1,33 +1,59 @@
-# =============== #
-# Unity generated #
-# =============== #
-Temp/
-Obj/
-UnityGenerated/
```

Comment:
UnityGenerated is not in the new file, should it not be ignored?

Reply:
Temp/ is replaced with lower case temp/
Obj/ is replaced with lower case obj/
UnityGenerated is not something that we need in our project and can be removed.